catalog/Developer_Tools/Benchmarking.yml

```diff
@@ -3,6 +3,7 @@ description:
 projects:
   - bench_press
   - benchmarkable
+  - benchmark-ips
```

Comment:
I've just noticed that the [tests failed](https://travis-ci.org/rubytoolbox/catalog/builds/339077036?utm_source=github_status&utm_medium=notification) because this list needs to be sorted alphabetically.